_includes/parse-server/development.md

```diff
@@ -14,6 +14,8 @@ You can then link the parse-server module to the cloned repo and run `npm instal
 
 ```sh
 npm link parse-server path/to/cloned/repo
+```
+```sh
```

Comment:
Done. (This is a personal opinion: I did split the code, because feels weird, to see more than 1 line of code in the same block, usually the pages included the button to copy the code, and having multiple lines (or the $ prefix) makes the button useless)

Reply:
That's a good point, not sure if we have such a copy/page button in our widget, but in any case I'd align with the test of the docs - if CLI commands are split into separate code blocks, then we can do that here too for consistency.

Reply:
Yes, you have the copy button, I saw in some places the code is not split, combined looks nicer. (I have an internal fight about right-thing/functionality vs aesthetics.) 
I am not 100% sure right now, but I think if I copy the combined block and paste it in the terminal, executes the first line only or uses the rest of the lines as parameter or something weird happens.., and I remember having some drama in the past thinking that I run all the lines.

Also some blocks use **sh** others **bash** and others nothing **```**,  also inconsistency using **js** and **javascript**

(I am not trying to convince you or whatever... just my 2c)